### PR TITLE
Fe605 my books delete

### PR DESCRIPTION
**Pull request by MatthewSpeaks**
Added MyLibrary component for user dashboard. Simple table that allows a user to view their books, select, and delete them from the database.

Future update to include:

    once Customers are setup and the backend controller is updated, this should dynamically update to display that specific users books.
    toggle functionality to toggle availability of the books without having to delete. Will need to finish handleToggle method and build a Book Controller method on the back end to receive those update requests.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -218,16 +219,74 @@ div {
...
   font-size: 1.1rem;
   cursor: pointer;
   font-weight: 600;
   transition: 0.2s;
 }
 
+table {
+  border-collapse: collapse;
+  width: 100%;
+}
```
Comment (dalpiad) [inline src/App.css]: The both table css lines and the td line are what mess up the home page.

File: src/App.css [hunk 2]
```diff
@@ -218,16 +219,74 @@ div {
...
+  background-color: #f2f2f2;
+  border: 1px solid #dddddd;
+  padding: 8px;
+  text-align: left;
+}
 
+td {
+  border: 1px solid #dddddd;
+  padding: 8px;
+}
```
Comment (dalpiad) [inline src/App.css]: The both table css lines and the td line are what mess up the home page.

File: src/App.css [hunk 3]
```diff
@@ -218,16 +219,74 @@ div {
...
+
+tbody {
+  border: 1px solid #dddddd;
+}
+
+table {
+  border-radius: 8px;
+  overflow: hidden;
+}
```
Comment (dalpiad) [inline src/App.css:292]: The both table css lines and the td line are what mess up the home page.

**Review by dalpiad — CHANGES_REQUESTED** [review]
Did some more digging and it is found where it is messing up the homepage.  You can probably add back the @emotion lines if you need them.  Just need to make the changes to the css file where I made the comments.

File: src/App.css (hunk 3 above)
Comment (MatthewSpeaks) [reply src/App.css:292]: I updated to put everything in CSS class. Looks to be working correctly now.

**Review by dalpiad — APPROVED** [review]
Looks good.

**Pull request merged**